### PR TITLE
Style Comments to Include Grey Background

### LINKED ISSUE #4204: Change frontend for Comments
Currently, the post action of comments looks as
image
we want it to change it to the following
image
In this issue we just want a grey div in background.
radio buttons are linked to all the tags of the note.
Refer #3932.

### PR DESCRIPTION
**Pull request by dinaelhanan**
Fixes #4204 

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by dinaelhanan** [comment]
@publiclab/reviewers please check this out!

**Comment by ValentinaTironi** [comment]
Don't forget the placeholder :)

**Comment by oorjitchowdhary** [comment]
@dinaelhanan Do you need any help? I wasn't able to submit a patch on time.. So I can help you out here maybe

**Comment by SidharthBansal** [comment]
@ValentinaTironi is helping her out. @oorjitchowdhary you can work out on the other issues in the meantime. Please don't claim more @oorjitchowdhary. I will advice you to first get all your pending rewards and after that you can continue to work on the existing issues.

**Comment by ValentinaTironi** [comment]
How is going on this @dinaelhanan?

**Comment by dinaelhanan** [comment]
@ValentinaTironi good thanks! I just need help on here first:
https://github.com/publiclab/plots2/pull/4225#discussion_r240214354

**Comment by SidharthBansal** [comment]
Dina I think you should complete this first. Its much smaller than other tasks.

**Comment by SidharthBansal** [comment]
Please also provide the screenshots

**Comment by dinaelhanan** [comment]
@SidharthBansal OK I will. I am having issues with adding a placeholder however

**Comment by SidharthBansal** [comment]
https://codein.withgoogle.com/dashboard/tasks/6008252891398144/
Here is the task for both of you.

**Comment by SidharthBansal** [comment]
What issue?

**Comment by dinaelhanan** [comment]
@SidharthBansal I do not know how to add the placeholder

**Comment by gauravano** [comment]
Please provide screenshot of the color change only. Placeholder is already there, I guess something's wrong with CSS which is preventing it to display. Thanks!

**Comment by dinaelhanan** [comment]
@gauravano oh ok thanks! I'm having trouble running this on c9 though, I can't seem to get it to show up and I keep getting redirected to localhost/user_sessions?return_to=notes/admin/blahblahblah
please help

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -54,7 +54,7 @@
     </div>
   </div>
 
-  <div class="well col-md-11" id="preview" style="background:white;display: none">
```
Comment (gauravano) [inline app/views/comments/_form.html.erb]: This is preview tab and not comment form.

Comment (gauravano) [reply app/views/comments/_form.html.erb]: Which means you are targetting wrong div.

Comment (dinaelhanan) [reply app/views/comments/_form.html.erb]: oh no.. thanks for pointing that out!

**Comment by dinaelhanan** [comment]
@gauravano OK I've changed it and will try running it on c9

**Comment by SidharthBansal** [comment]
Please provide the screenshots

**Comment by dinaelhanan** [comment]
@SidharthBansal it's not working on c9, can someone run it on their local version? I will set up my local version and use that from now on. Thanks!

**Comment by dinaelhanan** [comment]
@SidharthBansal @publiclab/reviewers done! 
screenshot from 2018-12-10 14-11-06

**Comment by gauravano** [comment]
Hey @dinaelhanan here's some modification done by me. What do you think about this design?

test_design

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -12,7 +13,7 @@
     <%= render :partial => "editor/toolbar" %>
   <% end %>
 
-  <div class="form-group dropzone">
+  <div class="form-group dropzone" style="background-color:grey;">
```
Comment (gauravano) [inline app/views/comments/_form.html.erb]:
```suggestion
  <div class="form-group dropzone">
```

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -54,9 +55,8 @@
     </div>
   </div>
 
-  <div class="well col-md-11" id="preview" style="background:white;display: none">
+  <div class="well col-md-11" id="preview" style="display: none">
```
Comment (gauravano) [inline app/views/comments/_form.html.erb]:
```suggestion
  <div class="well col-md-11" id="preview" style="background: white; display: none;">
```

File: app/views/comments/_form.html.erb [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<div style="background-color:#DCDCDC;border: 1px solid #bbb;border-bottom: 0;padding: 10px;">
```
Comment (gauravano) [inline app/views/comments/_form.html.erb]:
```suggestion
<div style="background-color:#f8f8f8; border: 1px solid #e7e7e7;padding: 18px;">
```

**Comment by gauravano** [comment]
@dinaelhanan this task would be complete just few changes.

File: app/views/comments/_form.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (dinaelhanan) [reply app/views/comments/_form.html.erb]: Oh we don't woops!

File: app/views/comments/_form.html.erb (hunk 4 above)
Comment (dinaelhanan) [reply app/views/comments/_form.html.erb]: OK I do like that design better, thanks!

File: app/views/comments/_form.html.erb [hunk 5]
```diff
@@ -54,9 +55,8 @@
     </div>
   </div>
 
-  <div class="well col-md-11" id="preview" style="background:white;display: none">
```
[the remark this answers is not in the document]
Comment (dinaelhanan) [reply app/views/comments/_form.html.erb]: Ya I know but I previously changed this to grey so I had to change it back

**Comment by dinaelhanan** [comment]
@gauravano I love that design! I tried to resolve all requests. Thanks!

**Comment by jywarren** [comment]
awesome! go ahead and merge, Gaurav, thank you!

**Comment by gauravano** [comment]
Just waiting to complete test @jywarren !

**Comment by gauravano** [comment]
Merged :tada: :smiley: . Awesome work @dinaelhanan!

**Comment by SidharthBansal** [comment]
Well done both of you.

**Pull request merged**